news/2025/2025-01-27-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,222 @@
+
+## Release schedule
+
+### ⏳ Star rating recalculation (24 - 48 hours)
+
+To begin, we will be recalculating the star rating of every beatmap across every game mode.
+
+This will be done live, so as the new SR values are applied to beatmaps, new scores arriving will get calculated using them. As a result, the PP you achieve may slightly deviate from expectations. Any PP values which don't match will be fixed in the following steps.
+
+Importantly, **the old PP algorithm will still be used, so the PP values achieved will not be final**.
+
+### ⏳ Freeze global rank graph updates
+
+We are about to reprocess every score in existence. Before doing this, we need to disable user rank updates. This means your current global / country rank (and graph) will be frozen in time until we're done reprocessing things.
+
+That said, if you play, your **total PP** will still immediately be updated.
+
+If we didn't do this, users would see their rank jump all over the place, as we have no way of ensuring every score PP and user total PP are updated all at once.
+
+### ⏳ Reprocess performance values of all scores (3 - 10 days)
+
+We now need to reprocess over 3 billion scores (this means **any scores you can currently see on the website** will get a new PP value, including scores set on osu!(stable) and osu!(lazer)). This is the most time-consuming part of the deploy process.
+
+During this period, scores in "best performance" may look to be out of order or not be visible at all. Also, **your global rank might jump around** momentarily as not all users will be recalculated at once.
+
+### ⏳ Reprocess total PP values for all users (12 hours)
+
+### ⏳ Re-enabling of global rank history updates
+
+Rank history graphs will be enabled and updated again. At this point, all users' global leaderboard ranks will be stable going forward.
+
+### ⏳ Reindexing (2 - 3 days)
+
+This will fix scores being out of order on profiles under "best performance", and in some rare cases not being displayed at all.
+
+## osu!
+
+### Acute angle bonus rework
+
+The acute angle bonus is a bonus inside of aim calculations for BPMs greater than 300. It is used to reward fast, difficult aim patterns.
+
+This bonus should now be more fair towards difficult patterns, while some overweighted patterns such as repeated angles are now awarded less of this bonus, thanks to a [set](https://github.com/ppy/osu/pull/30902) [of](https://github.com/ppy/osu/pull/31245) [various](https://github.com/ppy/osu/pull/31320) [changes](https://github.com/ppy/osu/pull/31566) proposed by [StanR](https://osu.ppy.sh/users/7217455) (with some contributions by [tsunyoku](https://osu.ppy.sh/users/11315329)).
+
+There was also a change to uncap velocity constraints that used to exist on the bonus allowing for some very hard *acute*-angled jumps in high BPM beatmaps to receive a greater buff.
+
+Some examples of beatmaps buffed by this change:
```

Comment:
It should be better clarified that those are examples for High BPM aim changes as a whole, not just the "uncap velocity constraints"

Reply:
bf50ad565d38c7aa8c89b634018c451b1c4ea91f